MusicTaste/MusicTaste/View Controllers/MatchmakingViewController.m

```diff
@@ -81,5 +82,14 @@ -(void)getMatchesDictionary{
     }];
 }
 
-
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
```

Comment:
Added prepareforsegue to transition information of the User to detailsViewController